test/test_conditional_feature_importance.py

```diff
@@ -569,3 +570,85 @@ def test_groups_warning(self, data_generator):
             " number of features for which importance is computed: 4",
         ):
             cfi.importance(X, y)
+
+
+@pytest.mark.parametrize(
+    "n_samples, n_features, support_size, rho, seed, value, signal_noise_ratio, rho_serial",
+    [(10, 3, 1, 0.2, 0, 1.0, 1.0, 0.0)],
+    ids=["high level noise"],
+)
+@pytest.mark.mpl_image_compare
+def test_cfi_plot(data_generator):
+    """Test CFI plot function"""
+    X, y, _, _ = data_generator
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.5, random_state=0
+    )
+    fitted_model = LinearRegression().fit(X_train, y_train)
+    cfi = CFI(
+        estimator=fitted_model,
+        imputation_model_continuous=LinearRegression(),
+        random_state=0,
+    )
+    # Make the plot independent of data / randomness to test only the plotting function
+    cfi.importances_ = np.arange(10)
+    fig, ax = plt.subplots(figsize=(6, 3))
+    ax = cfi.plot_importance(ax=ax)
+    return fig
+
+
+@pytest.mark.parametrize(
+    "n_samples, n_features, support_size, rho, seed, value, signal_noise_ratio, rho_serial",
+    [(10, 3, 1, 0.2, 0, 1.0, 1.0, 0.0)],
+    ids=["high level noise"],
+)
+@pytest.mark.mpl_image_compare
+def test_cfi_plot_2d_imp(data_generator):
+    """Test CFI plot function"""
+    X, y, _, _ = data_generator
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.5, random_state=0
+    )
+    fitted_model = LinearRegression().fit(X_train, y_train)
+    cfi = CFI(
+        estimator=fitted_model,
+        imputation_model_continuous=LinearRegression(),
+        random_state=0,
+    )
+    # Make the plot independent of data / randomness to test only the plotting function
+    cfi.importances_ = np.stack(
```

Comment:
This is not realist cases, actually.

Reply:
What we want to test here is the behavior of the plot function, not the importance. So I keep the importance fixed. Changes in the behavior of the importance method (for instance, changes in the random generator) should not trigger this test.

Reply:
Discussed in #449